hack/generated/controllers/crd_cosmosdb_test.go

```diff
@@ -18,13 +17,9 @@ import (
 func Test_CosmosDB_CRUD(t *testing.T) {
 	t.Parallel()
 
-	g := NewGomegaWithT(t)
-	ctx := context.Background()
-	testContext, err := testContext.ForTest(t)
-	g.Expect(err).ToNot(HaveOccurred())
+	testContext := testContext.ForTest(t)
```

Comment:
I think you could go the whole hog and call the local `tc`. (But now I've got the Top Cat theme tune stuck in my head.)

Reply:
that’s a good suggestion because the `Expect` calls are now very clunky